velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -125,37 +125,37 @@ const char* PageReader::readBytes(int32_t size, BufferPtr& copy) {
 
 const char* FOLLY_NONNULL decompressLz4AndLzo(
     const char* compressedData,
-    BufferPtr& uncompressedData,
+    BufferPtr& decompressedData,
     uint32_t compressedSize,
-    uint32_t uncompressedSize,
+    uint32_t decompressedSize,
```

Comment:
This should be uncompressedSize, which means the size before the data was compressed